stats/src/main/java/io/airlift/stats/cardinality/BiasCorrection.java

```diff
@@ -15,12 +15,13 @@
 
 // These tables are generated empirically by running several thousand experiments and computing the average error for each cardinality
 // TODO: check in the code for generating the tables
+@SuppressWarnings("MutablePublicArray")
```

Comment:
Can this be moved to the field that causes the problem?

Reply:
moved